scripts/local_db_setup.sh

```diff
@@ -18,4 +18,5 @@ docker run \
   -e POSTGRES_USER=$(cat secrets/db.user) \
   -e POSTGRES_DB=$(cat secrets/db.name) \
   -p $(cat secrets/db.port):5432 \
-  -d postgres:13
\ No newline at end of file
+  -d postgres:13 \
+  postgres -N 200
```

Comment:
can we also do this for:
https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/e2e71a395b032a27fa562ec3a206604196ea5204/docker/Dockerfile_template#L86 and 
https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/e2e71a395b032a27fa562ec3a206604196ea5204/docker/Dockerfile_template_mocked#L77 which are used for the midstream PR checks run

Reply:
Updated in 619c98528a5cf4ce9ef3958904de1692dcb90f43

Reply:
@machi1990  `CMD ["postgres", "-N", "200"]` didn't work when i tried it locally :disappointed: It started the database but didn't apply the max connection configuration. 

Instead, I added a sql command to update this in the db setup sql script which is used in both Dockerfile_template and Dockerfile_template_mocked. 

See cd6deb1d40a7ff9148fd55f35e635dd1dd577030

Reply:
Thanks @JameelB 

I am happy with the alternative solution. It's even better than the duplication in both docker template files

Reply:
Max connections config of db in pr check container: 
![image](https://user-images.githubusercontent.com/9078522/188131606-8c4dd530-cf42-4b18-8d57-51dc9a686063.png)